app/Locale/French.php

```diff
@@ -172,23 +173,23 @@ public function get_locale(): array
 			'ALBUM_SHR_YES' => 'Oui',
 			'ALBUM_SHR_NO' => 'Non',
 			'ALBUM_PUBLIC' => 'Public',
-			'ALBUM_PUBLIC_EXPL' => 'Des utilisateurs anonymes peuvent accéder à cet album, sous réserve des restrictions suivantes.',
+			'ALBUM_PUBLIC_EXPL' => 'Les utilisateurs anonymes peuvent accéder à cet album, sous réserve des restrictions suivantes.',
```

Comment:
Sorry :smiling_face_with_tear: I have never understood when the French use "des" or "les". :joy: